support-models/src/main/scala/com/gu/support/catalog/Product.scala

```diff
@@ -212,224 +218,73 @@ case object Paper extends Product {
 }
 
 case object GuardianWeekly extends Product {
-  private def productRatePlan(
+  private def domestic(
     id: String,
     billingPeriod:
     BillingPeriod,
```

Comment:
actually typo already existed 
```suggestion
    billingPeriod: BillingPeriod,
```